commons/src/main/java/fr/free/nrw/commons/upload/APICalls.java

```diff
@@ -0,0 +1,169 @@
+package fr.free.nrw.commons.upload;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.android.volley.Cache;
+import com.android.volley.NetworkResponse;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.HttpHeaderParser;
+import com.android.volley.toolbox.JsonRequest;
+import com.android.volley.toolbox.Volley;
+
+import com.google.gson.Gson;
+
+
+import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class APICalls {
```

Comment:
Obviously very minor but I recommend using camelCasing for initialisms too. So something like MwVolleyApi?